src/universal/config/myarea-datasets.ts

```diff
@@ -71,28 +79,28 @@ export const DATASETS: DatasetCategories = {
         title: 'Laadpalen',
         filters: {
           snellader: {
-            title: 'Snellader',
+            title: laadpaalValueConfig.snellader,
             valueConfig: {
               True: { title: 'Ja' },
               False: { title: 'Nee' },
             },
           },
           CHADEMO: {
```

Comment:
Het zou mooi zijn als je de Connector opties Onder een kopje oid kan zetten. Je kunt hiervoor bv een optionele optie "group" oid toevoegen (denk aan <fieldset label="Connectors"><input /><input /></fieldset>).

```
{
  groups: {
    group1: { title: 'Groep 1' },
    group2: { title: 'Groep 2' }
  },
  filters: {
    filter1: {
      group: 'group1'
    },
    filter2: {
      group: 'group1'
    }
    filter3: {
      group: 'group2'
    }
  }
}
```

Dergelijke configuratie zou je kunnen gebruiken om in de UI een setje filters visueel te groeperen. Misschien kan Bram een design maken als je niet weet hoe.

Verder, mocht je die Ja/Nee opties echt niet trekken, zou je die Ja/Nee opties, ui wise, ook kunnen Weergeven in een switch oid (probeer eens een set radio buttons om te kijken wat voor resultaten dat geeft) bv https://codepen.io/nickbottomley/pen/ndqVQP.

Reply:
maak ik een story voor aan!